test/findRoute.test.js

```diff
@@ -49,6 +49,22 @@ test('findRoute should return an immutable route to avoid leaking and runtime ro
   t.same(route.params, { artistId: ':artistId' })
 })
 
+test('findRoute should return null when when url is not passed', t => {
```

Comment:
this test for or use case of empty string [here](https://github.com/fastify/fastify/pull/5102/files#diff-cf6996cafd1df0bc168a5c5f3a7470f11fc3aea9f908091d5dd6be2e7b30a2c3R166)